x/authz/module/autocli.go

```diff
@@ -51,13 +50,25 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 			Service:              authzv1beta1.Msg_ServiceDesc.ServiceName,
 			EnhanceCustomCommand: true,
 			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod: "Exec",
+					Use:       "exec [msg-json-file] --from [grantee]",
+					Short:     "Execute tx on behalf of granter account",
+					Example:   fmt.Sprintf("$ %s tx authz exec msg.json --from grantee\n $ %[1]s tx bank send [granter] [recipient] [amount] --generate-only | jq .body.messages > msg.json && %[1]s tx authz exec msg.json --from grantee", version.AppName),
```

Comment:
There is actually a small behavior change, but I think it is minimal. Notice the example difference.